ui/admin/app/components/form/onboarding/index.js

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormOnboardingComponent extends Component {
+  // =properties
+  @tracked targetAddress;
+  @tracked targetAddressInValid = false;
```

Comment:
I think these could have been getters that check the `targetAddress` or `targetPort` values to determine if they're valid

Reply:
Since this comment could be related to the one above, let's discuss in our pod stand up zoom call.

Reply:
@ZedLi Refactored to a simplified way 😃 Thanks for the help!